### PR TITLE
Add language clarifying VP is short-lived

### LINKED ISSUE #937: Presentations SHOULD expire
In general guidance on presentations is poor.

My understanding of them is that they are meant to be extremely short lived, and bound to a challenge provided by a verifier.

Therefore, they should expire in ... minutes?

If they don't expire quickly, the value of the subject / holder arguments seems less useful.

The core data model should define requirements for presentations better, and the "securing formats" should enforce them.

### PR DESCRIPTION
**Pull request by brentzundel**
Fixes #937 

This PR uses language suggested in #937 to clarify that VPs are intended to be short-lived.

***
<a href="https://pr-preview.s3.amazonaws.com/brentzundel/vc-data-model/pull/1164.html" title="Last updated on Jun 23, 2023, 9:46 PM UTC (116a7e0)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/1164/329d6f8...brentzundel:116a7e0.html" title="Last updated on Jun 23, 2023, 9:46 PM UTC (116a7e0)">Diff</a>

### CODE REVIEW

**Review by msporny — APPROVED** [review]

**Review by decentralgabe — APPROVED** [review]

**Review by andresuribe87 — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -1917,7 +1917,12 @@ <h3>Presentations</h3>
 <a>verifiable credentials</a> is a typical use of
 <a>verifiable presentations</a>.
         </p>
-
+        <p>
+<a>Verifiable presentations</a> are intended to be extremely short-lived, and
+bound to a challenge provided by a <a>verifier</a>. Details for accomplishing
+this depend on the securing mechanism, the transport protocol, and
+<a>verifier</a> policies.
+        </p>
```
Comment (awoie) [inline index.html:1925]:
```suggestion
<a>Verifiable presentations</a> are intended to be extremely short-lived, and
bound to a challenge provided by a <a>verifier</a>. Details for accomplishing
this depend on the securing mechanism, the transport protocol, and
<a>verifier</a> policies. Without additional requirements defined by the particular securing
mechanism or embedding protocol, a <a>Verifier</a> cannot generally assume that the author of the
<a>Verifiable Presentation</a> corresponds to the first <a>Holder</a> or <a>Subject</a> of the 
<a>Verifiable Credential</a>.
        </p>
```

**Review by awoie — COMMENTED** [review]
I'm ok with that but we should also add some more text that says that verifiers cannot assume any correlation between the VP and VC.

**Review by awoie — APPROVED** [review]
actually, i approve this PR as is since the PR is already an improvement and is focused on the longevity. I created a separate issue for what my concern was.

File: index.html (hunk 1 above)
Comment (awoie) [reply index.html:1925]: pls ignore the suggestion above, i created a separate issue.

Comment (Sakurann) [reply index.html:1925]: why? looks like a good clarification to me?

Comment (TallTed) [reply index.html:1925]: Some tweaks to @awoie's suggestion... Fixes #1165.
```suggestion
<a>Verifiable presentations</a> are intended to be extremely short-lived, and
bound to a challenge provided by a <a>verifier</a>. Details for accomplishing
this depend on the securing mechanism, the transport protocol, and
<a>verifier</a> policies. Unless additional requirements are defined by the
particular securing mechanism or embedding protocol, a <a>Verifier</a> cannot
generally assume that the <a>Verifiable Presentation</a> has any correlation
with the presented <a>Verifiable Credentials</a> beyond preservation of their
data integrity.
        </p>
```

Comment (awoie) [reply index.html:1925]: I agree that with @TallTed 's tweaks from above, this PR could also close #1165.

File: index.html [hunk 2]
```diff
@@ -1917,7 +1917,12 @@ <h3>Presentations</h3>
 <a>verifiable credentials</a> is a typical use of
 <a>verifiable presentations</a>.
         </p>
-
+        <p>
+<a>Verifiable presentations</a> are intended to be extremely short-lived, and
```
Comment (OR13) [inline index.html:1921]: This does not seem to align with the normative requirements we have in the spec.

File: index.html (hunk 1 above)
Comment (OR13) [reply index.html:1925]: Verifiable Credentials are not required to have any data integrity, proof is optional.

Additionally, `proof` does not secure the bytes so a presentation could end up securing different information if different securing mechanisms are used.

I think the sentence is better without the "beyond preservation of their data integrity." part.

File: index.html (hunk 2 above)
Comment (OR13) [inline index.html:1921]:
```suggestion
<a>Verifiable presentations</a> MAY be extremely short-lived, and
```

**Review by OR13 — CHANGES_REQUESTED** [review]
Use normative language regarding securing presentations.

Avoid asserting that credentials have integrity when there is no requirement that they do.

File: index.html [hunk 3]
```diff
@@ -1917,7 +1917,16 @@ <h3>Presentations</h3>
...
-
+        <p>
+<a>Verifiable presentations</a> MAY be extremely short-lived, and
+bound to a challenge provided by a <a>verifier</a>. Details for accomplishing
+this depend on the securing mechanism, the transport protocol, and
+<a>verifier</a> policies. Unless additional requirements are defined by the
+particular securing mechanism or embedding protocol, a <a>Verifier</a> cannot
+generally assume that the <a>Verifiable Presentation</a> has any correlation
+with the presented <a>Verifiable Credentials</a> beyond preservation of their
+data integrity.
```
Comment (brentzundel) [inline index.html:1928]:
```suggestion
<a>Verifiable presentations</a> MAY be extremely short-lived, and
bound to a challenge provided by a <a>verifier</a>. Details for accomplishing
this depend on the securing mechanism, the transport protocol, and
<a>verifier</a> policies. Unless additional requirements are defined by the
particular securing mechanism or embedding protocol, a <a>verifier</a> cannot
generally assume that the <a>verifiable presentation</a> has any correlation
with the presented <a>verifiable credentials</a>.
```

File: index.html [hunk 4]
```diff
@@ -1917,7 +1917,15 @@ <h3>Presentations</h3>
 <a>verifiable credentials</a> is a typical use of
 <a>verifiable presentations</a>.
         </p>
-
+        <p>
+<a>Verifiable presentations</a> MAY be extremely short-lived, and
```
Comment (brentzundel) [inline index.html:1921]:
```suggestion
<a>Verifiable presentations</a> SHOULD be extremely short-lived, and
```

**Comment by brentzundel** [comment]
I accepted your suggestions, then continued to tweak the language.

@OR13 @TallTed @awoie please re-review.

**Review by TallTed — APPROVED** [review]

**Review by dlongley — APPROVED** [review]

**Review by awoie — APPROVED** [review]
thanks, lgtm.

**Review by OR13 — APPROVED** [review]

**Comment by msporny** [comment]
Normative, multiple reviews, changes requested and made, no objections, merging.

File: index.html (hunk 2 above)
Comment (RieksJ) [reply index.html]: I disagree - see my [post](https://github.com/w3c/vc-data-model/issues/937#issuecomment-1265409715). Also, no example has been provided why VPs are intended to be (extremely) short lived, as suggested by @Sakurann s [comment](https://github.com/w3c/vc-data-model/issues/937#issuecomment-1387552819). So, lacking any substantiation for the assertion that presentations are intended to be extremely short-lived, and noting that the standard cannot know the intentions of holders regarding the presentations they create,I suggest to not merge this.

**Pull request merged**